testing/Makefile

```diff
@@ -0,0 +1,18 @@
+.DEFAULT_GOAL := help
+
+SUDO          ?= sudo
+INSTALLOPTS   ?= -Su
+
+VOID_PKGS     = perl-boolean perl-Sort-Versions lsof yq-go qemu
```

Comment:
do we need all of qemu? that includes all user and system emulators

Reply:
Good call. I've narrowed it down to what's strictly needed to create/start an amd64 guest.